extensions/WSLExtension/Contracts/IWslManager.cs

```diff
@@ -52,4 +53,10 @@ public interface IWslManager
     /// This is a wrapper for <see cref="IWslServicesMediator.IsDistributionRunning(string)"/>
     /// </summary>
     public bool IsDistributionRunning(string distributionName);
+
+    /// <summary> Installs the WSL kernel package from the Microsoft store if it is not already installed. </summary>
+    public Task InstallWslKernelPackageAsync(Action<string>? statusUpdateCallback, CancellationToken cancellationToken);

```

Comment:
Why not a general class to install packages? That way it can be reused in Dev Home.

Reply:
we have one in Dev Home and that's what I use in that method. See the `DevHome.Services` project. I use the `MicrosoftStoreService` to download the package